core/src/main/scala/spire/std/byte.scala

```diff
@@ -59,7 +59,7 @@ trait ByteOrder extends Order[Byte] {
 
 trait ByteSigned extends Signed[Byte] with ByteOrder {
   override def signum(a: Byte): Int = java.lang.Integer.signum(a)
-  override def abs(a: Byte): Byte = (if (a < 0) -a else a).toByte
+  override def abs(a: Byte): Byte = (if (a < 0) -a else a.toInt).toByte
```

Comment:
This doesn't look good. I think @armanbilge found a way to have just 1 `toByte` call. Same for `Short` and possibly other numeric types.

Reply:
As a reference
https://github.com/lampepfl/dotty/issues/12985

Reply:
I understand the problem. I believe we agreed on the following.

```scala
def abs(a: Byte): Byte = if (a < 0) (-a).toByte else a
```

Reply:
Right, I fixed this now in `byte` and `short`